components/Tabs/src/private/MDCItemBar.m

```diff
@@ -109,14 +112,19 @@ - (void)commonItemBarInit {
   UICollectionView *collectionView =
       [[UICollectionView alloc] initWithFrame:self.bounds collectionViewLayout:_flowLayout];
   collectionView.backgroundColor = [UIColor clearColor];
-  collectionView.clipsToBounds = YES;
+  collectionView.clipsToBounds = NO;
   collectionView.scrollsToTop = NO;
   collectionView.showsHorizontalScrollIndicator = NO;
   collectionView.showsVerticalScrollIndicator = NO;
+
+#if defined(__IPHONE_11_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_11_0)
+  if (@available(iOS 11.0, *)) {
+    collectionView.contentInsetAdjustmentBehavior = UIScrollViewContentInsetAdjustmentAlways;
```

Comment:
We risk introducing vertical scrolling in our tabs if we do this. If the client lays out the tab bar smaller than its sizeThatFits, instead of the item bar just being clipped it will become vertically scrollable. Thoughts on that? I don't think scrolling is necessarily strictly worse than clipping, it just feels a little weird.

Reply:
Both sound like a bug on their end tbh. What if we use UIScrollViewContentInsetAdjustmentScrollableAxes instead?

Reply:
I think just automatic (the default) will do what we need here, it'll scroll horizontally and not vertically.

When I say clipping, I simply mean if we make the containing view smaller than our UICollectionView with clipsToBounds set to YES (set above) it'll clip. Thats I think the behavior we would want.